block_iagora.php

```diff
@@ -48,7 +49,14 @@ public function get_content() {
 
         $this->content = new stdClass();
         $this->content->footer = '';
-        $this->content->text = '';
+
+        $iframe_url = isset($this->config->iframe_url) ? $this->config->iframe_url : '';
+
+        if (empty($iframe_url)) {
+            $this->content->text = get_string('noiframe_url', 'block_iagora');
```

Comment:
```suggestion
            $this->content->text = get_string('noiframeurl', 'block_iagora');
```
This seems to be the Moodle naming convention.

Also, this variable (`noiframeurl`) is missing the English translation string in the `lang/en/block_iagora.php` file.
For example:
```php
 $string['noiframeurl'] = 'No iframe URL';
```